docs/user-journeys/users/application-developer/foundational.md

```diff
@@ -102,12 +173,16 @@ The Kubernetes documentation is rich in detail. Here's a curated list of resourc
 
```

Comment:
Let's include the [Kubernetes Basics](https://kubernetes.io/docs/tutorials/kubernetes-basics/) tutorial in this list.